test/framework/src/main/java/org/elasticsearch/test/hamcrest/ElasticsearchAssertions.java

```diff
@@ -343,26 +315,18 @@ public static void assertFailures(SearchRequestBuilder searchRequestBuilder, Res
 
     public static void assertNoFailures(BroadcastResponse response) {
         assertThat("Unexpected ShardFailures: " + Arrays.toString(response.getShardFailures()), response.getFailedShards(), equalTo(0));
-        assertVersionSerializable(response);
     }
 
     public static void assertAllSuccessful(BroadcastResponse response) {
         assertNoFailures(response);
         assertThat("Expected all shards successful",
                 response.getSuccessfulShards(), equalTo(response.getTotalShards()));
-        assertVersionSerializable(response);
     }
 
     public static void assertAllSuccessful(SearchResponse response) {
         assertNoFailures(response);
         assertThat("Expected all shards successful",
                 response.getSuccessfulShards(), equalTo(response.getTotalShards()));
-        assertVersionSerializable(response);
-    }
-
-    public static void assertSearchHit(SearchHit searchHit, Matcher<SearchHit> matcher) {
-        assertThat(searchHit, matcher);
-        assertVersionSerializable(searchHit);
```

Comment:
This method was inlined since it was now just passing its arguments to `assertThat`

Reply:
makes sense 🌮